lib/Db/CardMapper.php

```diff
@@ -141,6 +141,11 @@ public function findAllByStack($stackId, $limit = null, $offset = null) {
 		return $this->findEntities($sql, [$stackId], $limit, $offset);
 	}
 
+	public function findAllWithDue() {
+		$sql = 'SELECT * from `*PREFIX*deck_cards` WHERE duedate IS NOT NULL AND NOT archived AND deleted_at = 0';
```

Comment:
:warning: This will return also cards of other users.

Reply:
So we either need a larger query with joining the board and acl tables together to get only the cards which are part of the users boards or we need to iterate over all users boards an fetch the cards per board.